alpha/lib/model/KuserToUserRolePeer.php

```diff
@@ -46,12 +61,34 @@ public function getRootObjects(IRelatedObject $object)
 			$object->getUserRole(),
 		);
 	}
-
+	
 	/* (non-PHPdoc)
 	 * @see IRelatedObjectPeer::isReferenced()
 	 */
 	public function isReferenced(IRelatedObject $object)
 	{
 		return false;
 	}
+	
+	/**
+	 * Get object by kuser and appGuid
+	 *
+	 * @param $kuserId
+	 * @param $appGuid
+	 *
+	 * @return KuserToUserRole|null
+	 * @throws PropelException
+	 */
+	public static function getByKuserIdAndAppGuid($kuserId, $appGuid)
+	{
+		$c = new Criteria();
+		$c->addAnd(self::KUSER_ID, $kuserId, Criteria::EQUAL);
+		$c->addAnd(self::APP_GUID, $appGuid, Criteria::EQUAL);
```

Comment:
Since you disable the default criteria please add the partnerId to the query as well

Reply:
We do not have partnerId on this bridge table.
But kuserId + appGuid is an index - so should be OK.